screwdriver.yaml

```diff
@@ -8,21 +6,21 @@ shared:
 
 jobs:
     main:
+        requires: [~pr, ~commit]
         steps:
             - get: go get -t ./...
             - vet: go vet ./...
             - gofmt: "find . -name '*.go' | xargs gofmt -s -w"
             - test: go test ./...
-            - build: go build -a -o meta
+            - build: go build -a -o $SD_ARTIFACTS_DIR/meta
 
-    deploy:
+    publish:
+        requires: [main]
         steps:
             - setup-ci: git clone https://github.com/screwdriver-cd/toolbox.git ci
             - get: go get -t ./...
-            - gofmt: "find . -name '*.go' | xargs gofmt -s -w"
```

Comment:
why are we removing gofmt?

Reply:
This can change the code, and if there is new changes, go releaser will not do the publish.

Reply:
I see 👍